src/codegen.cpp

```diff
@@ -1282,27 +1282,7 @@ jl_value_t *jl_static_eval(jl_value_t *ex, void *ctx_, jl_module_t *mod,
             jl_value_t *f = jl_static_eval(jl_exprarg(e,0),ctx,mod,sp,ast,sparams,allow_alloc);
             if (f && jl_is_function(f)) {
                 jl_fptr_t fptr = ((jl_function_t*)f)->fptr;
-                if (fptr == &jl_apply_generic) {
```

Comment:
this was support for a pre 0.1 deprecation from when ccall could only handle constant pointers. i removed jl_base_module, so i figured it was sensible to remove this special case also.